✨Breadcrumbs #139

kosterm14 opened the pull request:
## What does this change?
Ervoor gezorgd dat de breadcrumbs ook laten zien van welke pagina je komt als je bijvoorbeeld vanuit leeslijst naar uitleningen gaat. Voorheen zag je het alleen als je vanuit home naar een andere pagina ging.

Ook geven de breadcrumbs nu een max van de laatste drie pagina's weer zodat het niet de hele pagina inneemt. Dit is aangegeven met punten zodat de gebruiker weet dat er eigenlijk meer zijn.

Resolves issue #139 

<!-- A PR should have enough detail to be understandable far in the future. e.g what is the problem/why is the change needed, how does it solve it and any questions or points of discussion. Prefer copying information from a GitHub issue over linking to it; the card may not always exist and reviewers may not have access to the board. -->

Het is moeilijk om een live site link te geven van een specifieke pagina omdat je ook echt op meerdere pagina's moet drukken voordat de breadcrumbs werken

## How Has This Been Tested?

- [ ] User test
- [ ] Accessibility test
- [ ] Performance test
- [ ] Responsive Design test
- [ ] Device test
- [ ] Browser test

## Images

<!-- Usually only applicable to UI changes, what did it look like before and what will it look like after? -->

Before:
Maakt niet uit waar je vandaag komt er staat altijd home>current page

![image](https://github.com/fdnd-agency/oba/assets/61830362/b395bef3-6f8b-4bee-9fbe-2e8eefd65fd8)

After:
Er staat nu ook echt waar je vandaag komt met ... om aan te geven dat er meer dan 3 recente pagina's zijn (indien toepasselijk)

![image](https://github.com/fdnd-agency/oba/assets/61830362/859bc6a4-5208-45e6-99dc-e0e6893a52d6)

## How to review

<!-- Provide instructions to help others verify the change. This could take the form of "On PROD, do X and witness Y. On this branch, do X and witness Z. " -->

1. Navigeer vanuit de home page naar een andere pagina doormiddel van het nav menu

![image](https://github.com/fdnd-agency/oba/assets/61830362/a884b47f-722f-4573-abba-260af7e01605)
![image](https://github.com/fdnd-agency/oba/assets/61830362/9af090d6-2811-478f-b0dd-702786d29a44)

2. Klik op een andere route
3. Klik nogmaals op een andere route (niet home)
4. Je zult nu zien dat er 3 breadcrumbs staan die aangeven van welke pagina je kwam om hier te komen (dit was niet zo in de oude versie daar zou altijd home>current page gestaan hebben zelfs als je niet vanuit home kwam)
5. Als je nu nogmaals op een andere route drukt zul je zien dat er ... voor de minst recente staat om aan te geven dat er meer zijn maar er zullen altijd maar 3 weergeven worden.

kosterm14 committed:
✨Breadcrumbs #139

Ervoor gezorgd dat de breadcrumbs ook laten zien van welke pagina je komt als je bijvoorbeeld vanuit leeslijst naar uitleningen gaat. Voorheen zag je het alleen als je vanuit home naar een andere pagina ging.

Ook geven de breadcrumbs een max van de laatste drie pagina's weer zodat het niet de hele pagina inneemt.

joostf commented:
@kosterm14 ik snap dat een specifieke link niet heel nuttig is, maar de algemene live link waar deze verandering te bekijken is wel. Voeg je die nog even toe?!

kosterm14 commented:
> @kosterm14 ik snap dat een specifieke link niet heel nuttig is, maar de algemene live link waar deze verandering te bekijken is wel. Voeg je die nog even toe?!

Ik dacht er aan om de link erbij te zetten maar de website staat live op https://mijn-oba.netlify.app/ op David's repo en dat is een oude versie die de breadcrumbs niet heeft. De nieuwe branch heeft op de oba repo geen live link toch?

kosterm14 committed:
chore: Update Svelte adapter to use Vercel adapter

This commit updates the Svelte adapter in the `svelte.config.js` file and the `package.json` file to use the `@sveltejs/adapter-vercel` version `5.3.0`. This change is made to improve the deployment process by using the Vercel adapter.

kosterm14 committed:
reinstall node/package.json

kosterm14 committed:
Update vite.config.js

ik doe dit omdat vercel mijn website niet wil deployen

Error: [commonjs--resolver] Failed to resolve entry for package "graphql-request". The package may have incorrect main/module/exports specified in its package.json.

ik krijg die error en ik probeer deze commit als fix

joostf commented:
Hi Marco,

Heb je via de mail gereplied? Ik zie je comment niet op GitHub verschijnen?

Omdat jullie nu een eigen fork hebben naar de agency repo voor de OBA, is het wellicht handig om een eigen live link te maken via bijvoorbeeld Netlify?!

👋
Joost

On 14 May 2024, at 09:35, MKCO ***@***.***> wrote:



@kosterm14<https://github.com/kosterm14> ik snap dat een specifieke link niet heel nuttig is, maar de algemene live link waar deze verandering te bekijken is wel. Voeg je die nog even toe?!

Ik dacht er aan om de link erbij te zetten maar de website staat live op https://mijn-oba.netlify.app/ op David's repo en dat is een oude versie die de breadcrumbs niet heeft. De nieuwe branch heeft op de oba repo geen live link toch?

—
Reply to this email directly, view it on GitHub<https://github.com/fdnd-agency/oba/pull/151#issuecomment-2109487842>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAIDGAF7F5U5TTE4DMCOB43ZCG5ETAVCNFSM6AAAAABHUBPJ7OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBZGQ4DOOBUGI>.
You are receiving this because you commented.Message ID: ***@***.***>

kosterm14 commented:
> Hi Marco, Heb je via de mail gereplied? Ik zie je comment niet op GitHub verschijnen? Omdat jullie nu een eigen fork hebben naar de agency repo voor de OBA, is het wellicht handig om een eigen live link te maken via bijvoorbeeld Netlify?! 👋 Joost On 14 May 2024, at 09:35, MKCO ***@***.***> wrote: @kosterm14<https://github.com/kosterm14> ik snap dat een specifieke link niet heel nuttig is, maar de algemene live link waar deze verandering te bekijken is wel. Voeg je die nog even toe?! Ik dacht er aan om de link erbij te zetten maar de website staat live op https://mijn-oba.netlify.app/ op David's repo en dat is een oude versie die de breadcrumbs niet heeft. De nieuwe branch heeft op de oba repo geen live link toch? — Reply to this email directly, view it on GitHub<[#151 (comment)](https://github.com/fdnd-agency/oba/pull/151#issuecomment-2109487842)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAIDGAF7F5U5TTE4DMCOB43ZCG5ETAVCNFSM6AAAAABHUBPJ7OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBZGQ4DOOBUGI>. You are receiving this because you commented.Message ID: ***@***.***>

Ja ik heb de hele tijd errors met het deployen op vercel en netlify dus ik had mijn comment verwijderd ik ga morgen op school om hulp vragen en dan de link erbij zetten sorry dat het zo lang duurt

joostf commented:
No problemo, je kan bij David kijken en ik kan je morgenmiddag ook even helpen!

On 14 May 2024, at 22:08, MKCO ***@***.***> wrote:



Hi Marco, Heb je via de mail gereplied? Ik zie je comment niet op GitHub verschijnen? Omdat jullie nu een eigen fork hebben naar de agency repo voor de OBA, is het wellicht handig om een eigen live link te maken via bijvoorbeeld Netlify?! 👋 Joost On 14 May 2024, at 09:35, MKCO @.> wrote: @kosterm14<https://github.com/kosterm14>https://github.com/kosterm14 ik snap dat een specifieke link niet heel nuttig is, maar de algemene live link waar deze verandering te bekijken is wel. Voeg je die nog even toe?! Ik dacht er aan om de link erbij te zetten maar de website staat live op https://mijn-oba.netlify.app/ op David's repo en dat is een oude versie die de breadcrumbs niet heeft. De nieuwe branch heeft op de oba repo geen live link toch? — Reply to this email directly, view it on GitHub<#151 (comment)<https://github.com/fdnd-agency/oba/pull/151#issuecomment-2109487842>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AAIDGAF7F5U5TTE4DMCOB43ZCG5ETAVCNFSM6AAAAABHUBPJ7OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBZGQ4DOOBUGI. You are receiving this because you commented.Message ID: @.>

Ja ik heb de hele tijd errors met het deployen op vercel en netlify dus ik had mijn comment verwijderd ik ga morgen op school om hulp vragen en dan de link erbij zetten sorry dat het zo lang duurt

—
Reply to this email directly, view it on GitHub<https://github.com/fdnd-agency/oba/pull/151#issuecomment-2111054902>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAIDGAAXO3FZYE5MHIOEYW3ZCJVNXAVCNFSM6AAAAABHUBPJ7OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMJRGA2TIOJQGI>.
You are receiving this because you commented.Message ID: ***@***.***>

kosterm14 committed:
chore: Update Svelte adapter to use Auto adapter
